core/trie/trie.go

```diff
@@ -11,8 +11,11 @@ import (
 	"github.com/NethermindEth/juno/core/crypto"
 	"github.com/NethermindEth/juno/core/felt"
 	"github.com/NethermindEth/juno/db"
+	"github.com/NethermindEth/juno/utils"
 )
 
+const globalTrieHeight = 251 // TODO(weiihann): this is declared in core also, should be moved to a common place
```

Comment:
Yeah, I noticed this also when working on snap-sync.
IMO should be single place which a public constant